airflow/models/baseoperator.py

```diff
@@ -1146,27 +1115,22 @@ def add_only_new(self, item_set: Set[str], item: str) -> None:
         else:
             item_set.add(item)
 
+    @property
+    def operator(self) -> "BaseOperator":
```

Comment:
One request. Is it possible to make this property a `List[BaseOperator]` instead? If we want `TaskGroup` to implement `TaskMixin`, it'll be difficult to decide what `operator` should be if it's a singular operator.

Reply:
I refactored it a bit so now its `roots` property - more in line with your initaill proposition